profile.md

```diff
@@ -234,7 +234,14 @@ The following properties are defined for usage within a Kubernetes deployment:
   Version: 2.14
 
   The unique identifier for the Kubernetes cluster from which the request
-  was sent. This property MUST be a non-empty string serialized as follows:
+  was sent. This property can be useful when a Service Broker tries to
+  call back to a Platform that is managing multiple Kubernetes clusters.
+  Typically, this value will be static for the lifetime of the cluster,
+  however, that is a Platform implementation choice. Brokers ought to take
```

Comment:
better punctuation?: Typically this value will be static for the lifetime of the cluster, however, that is a Platform implementation choice.

In general this has a lot of `can` and `will` and other feedback on other PRs has been that these words need to either be spec words or needs to be reworked to not be noisy

Reply:
I don't think they can be RFC2119 keywords since we're not mandating anything - just providing info. Not sure what you mean by "noisy"

Reply:
ok I fixed the punctuation issue

Reply:
shoot... one more: 

> Brokers ought to take care that while atypical, it is possible for the value to change over time.

should be 

> Brokers ought to take care that, while atypical, it is possible for the value to change over time.

Reply:
fixed